Gruntfile.js

```diff
@@ -26,8 +26,16 @@ module.exports = function(grunt) {
     concat: {
       options: {
         banner: '<%= meta.banner %>' + '\n' +
-          '(function ( window, angular, undefined ) {' + '\n',
-        footer: '})( window, window.angular );'
+        "(function (factory) {\n" +
+        "  if (typeof define === 'function' && define.amd) {\n" +
+        "    define(['exports', 'angular'], factory);\n" +
+        "  } else if (typeof module === 'object' && typeof module.exports === 'object') {\n" +
+        "    factory(module.exports, require('angular'));\n" +
+        "  } else if (root.hasOwnProperty('angular')) {\n" +
+        "    factory((root.exports || (root.exports = {})), root.angular);\n" +
```

Comment:
I think you should export it directly to the global object like:

```javascript
factory(this, this.angular)
```

Here's also an example of UMD definition which requires a return value of you module: https://github.com/umdjs/umd/blob/master/templates/returnExports.js